docs/release-notes.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Release Notes Workflow
+
+Release notes are captured during the development cycle from [PR titles (or body
+blocks)](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) using [release notes
+labels](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) with compliance across master and release- branches using the munger in [contrib/mungegithub](https://github.com/kubernetes/contrib/blob/master/mungegithub/README.md#submit-queue)
+on the main Kubernetes repository (and future repositories later).
+
+Releases are built and published by the anago tool in [this repo](https://github.com/kubernetes/release) with the release notes published in [kubernetes/CHANGELOG.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md).
+
+The automated release notes gathered in this way are considered complete
+for alpha, beta and official patch releases.
+
+For new branch, major milestones (1.X.0), the automated release
+note tooling provides a complete set of changes since the last major
+milestone release by referencing all alphas and betas for the minor .X. version.
+
+This bootstraps a milestone release in CHANGELOG.md with a template for
+use by the Release Czar and the subteam area leads to then further expand
+on the release notes with more detail as needed.
+
+A major milestone release (1.X.0) in CHANGELOG.md might look like:
+
+* Changes since v1.2.0
+   * 1.3.0-beta.2
```

Comment:
That's a fine question.  As a user I might find it useful to have the laundry list of changes left in their respective pre-releases, rather than viewing it (duplicated in the same CHANGELOG.md file) as a single [long list](https://github.com/david-mcmahon/release/blob/rn-1.3/relnotes-release-1.3.md).  But I really just want what most users want.